lib/msprime.h

```diff
@@ -273,6 +273,10 @@ typedef struct {
     double strength;
 } instantaneous_bottleneck_t;
 
+typedef struct {
```

Comment:
Looks like we don't actually need this because there's no parameters. We can delete this struct and its place in the union. (Time is stored already in demographic_event_t)

Reply:
👍